src/routes/data.rs

```diff
@@ -217,3 +218,16 @@ pub async fn retrieve_card(
 
     Ok(Json(card.try_into()?))
 }
+
+/// `/data/fingerprint` handling the creation and retrieval of card fingerprint
```

Comment:
we have currently only planned to use this API for cards, this shouldn't be in `/data` right?

Reply:
for both `/cards` and `/data` we are making use of same api handler